pkg/factory/risingwave_object_factory.go

```diff
@@ -1519,6 +1545,159 @@ func (f *RisingWaveObjectFactory) setupConnectorContainer(container *corev1.Cont
 	})
 }
 
+func (f *RisingWaveObjectFactory) argsForStandalone() []string {
+	return []string{
+		"--meta-opts=" + strings.Join([]string{
+			fmt.Sprintf("--listen-addr 127.0.0.1:%d", consts.MetaServicePort),
+			fmt.Sprintf("--advertise-addr 127.0.0.1:%d", consts.MetaServicePort),
+			fmt.Sprintf("--dashboard-host 0.0.0.0:%d", consts.MetaDashboardPort),
+			fmt.Sprintf("--prometheus-host 0.0.0.0:%d", consts.MetaMetricsPort),
+			"--backend $(RW_BACKEND)",
+			"--state-store $(RW_STATE_STORE)",
+			"--data-directory $(RW_DATA_DIRECTORY)",
+			"--config-path " + risingwaveConfigPath,
```

Comment:
Seems like the following options are missing:
- `--etcd-endpoints`
- `--data-directory`
I suppose they are set via environment variables?

Reply:
If they are set via env vars, the meta node embedded in the standalone process won't be able to see them though.

They have to be supplied explicitly.

Reply:
Oh actually thinking about it more. I guess it should work, albeit unintentionally. Because we still call the parser in meta, which should be able to check the environment for these values.

Just have to be careful not to use env vars with the same name across different nodes.

Reply:
I will add a test to verify this.

Reply:
Oh, you are right. I forgot them. Let me add it.

Reply:
Updated!